docker-compose-mysql8.yml

```diff
@@ -0,0 +1,19 @@
+---
+version: '2'
+services:
+  # To test with MariaDB, set FLEET_MYSQL_IMAGE to mariadb:10.6 or the like.
+  mysql:
+    image: ${FLEET_MYSQL_IMAGE:-mysql:8}
```

Comment:
Why not just use the other docker-compose but with `FLEET_MYSQL_IMAGE=mysql:8`?

Reply:
I also added another command line arg to the mysql command.